addons/purchase_requisition/models/purchase_requisition.py

```diff
@@ -160,17 +161,23 @@ def _onchange_product_id(self):
             self.schedule_date = self.requisition_id.schedule_date
 
     @api.multi
-    def _prepare_purchase_order_line(self, name, product_qty=0.0, price_unit=0.0, taxes_ids=False):
+    def _prepare_purchase_order_line(self, name, product_qty=0.0, price_unit=0.0, taxes_ids=False, partner_id=False, po=False):
         self.ensure_one()
         requisition = self.requisition_id
+        seller = self.product_id._select_seller(
+            partner_id=partner_id,
+            quantity=product_qty,
+            date=requisition.schedule_date or fields.Date.today(),
+            uom_id=self.product_id.uom_po_id)
+        date_planned = requisition.schedule_date or self.env['purchase.order.line']._get_date_planned(seller, po=po).strftime(DEFAULT_SERVER_DATETIME_FORMAT)
```

Comment:
`fields.Datetime` has helpers to convert between datetime object and the server format, there is no reason to invoke strftime manually.

Reply:
Hi @xmo-odoo.  Here's a commit to fix that.